### ISSUE TITLE
Missing #include when compiling with gcc-6

### ISSUE THREAD

**Issue by geisserf** [prost-planner/prost#130]
Reported by Mingxuan Li, reproduced by @geisserf .

Compiling the planner with gcc-6 gives the following error:

```
prost/src/rddl_parser/logical_expressions_includes/calculate_domain_as_interval.cc: In member function ‘virtual void prost::parser::StateFluent::calculateDomainAsInterval(const Domains&, const prost::parser::ActionState&, double&, double&)’:
prost/src/rddl_parser/logical_expressions_includes/calculate_domain_as_interval.cc:15:21: error: ‘numeric_limits’ was not declared in this scope
     assert(minRes = numeric_limits<double>::max());
                     ^
prost/src/rddl_parser/logical_expressions_includes/calculate_domain_as_interval.cc:15:36: error: expected primary-expression before ‘double’
     assert(minRes = numeric_limits<double>::max());
```

**Comment by geisserf** [comment]
I did not look into why this does not happen with newer gcc versions, but the underlying issue is that `<limits>` is not included at this part of the code.

**Comment by thomaskeller79** [comment]
Thanks for taking care of this, the issue has been resolved and merged in 565d6efbb9e3f259521067e37bbd282f8aa0fdf0
